app/models/user.rb

```diff
@@ -26,7 +26,8 @@
 #  email_notification_type      :integer          default("full_html_email")
 #  isactive                     :boolean          default(TRUE)
 #  last_activated_at            :datetime
-
+require 'open-uri'
+require 'elasticsearch/model'
```

Comment:
can we remove these requires if it is not used?

Reply:
open-uri is used by csv, removed elasticsearch

Reply:
thanks, looks good